### PR TITLE
improvement in Makefile.runtime and runtime.

### PR DESCRIPTION
**Pull request by bignaux**

### CODE REVIEW

**Comment by probonopd** [comment]
Thanks @bignaux but cannot merge yet since it is still crippling the 1024 bytes sections.
Compare https://travis-ci.org/probonopd/appimagetool/builds/168346010#L597-L615 with the master branch.

File: runtime.c [hunk 1]
```diff
@@ -348,7 +349,7 @@ main (int argc, char *argv[])
             "if you run it with the --appimage-extract option. \n"
             "See https://github.com/probonopd/AppImageKit/wiki/FUSE \n"
             "for more information";
-            notify(title, body, NULL); // 3 seconds timeout
+	    notify(title, body, 3000); // 3 seconds timeout
```
Comment (probonopd) [inline runtime.c]: Actually the NULL was correct and the comment was wrong here.

Comment (bignaux) [inline runtime.c]: NULL is not 0, was not correct anyway.

Comment (probonopd) [inline runtime.c]: Let's say "it did exactly what I wanted - display a GUI dialog rather than a notification" :)

File: Makefile.runtime [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# Add .upd_info and .sha256_sig sections
+embed: 1024_blank_bytes runtime
+	objcopy --add-section .upd_info=1024_blank_bytes \
+		--set-section-flags .upd_info=noload,readonly runtime
+	objcopy --add-section .sha256_sig=1024_blank_bytes \
+		--set-section-flags .sha256_sig=noload,readonly runtime
+	stat runtime
+
+# Now statically link against libsquashfuse_ll, libsquashfuse and liblzma
+# TODO: generate runtime in fonction of the compressor we choose to avoid embeded unncessary compression.
```
Comment (probonopd) [inline Makefile.runtime]: Please fix spelling "function"

File: runtime.c (hunk 1 above)
Comment (probonopd) [inline runtime.c]: Please make it 0

File: build.sh [hunk 3]
```diff
@@ -11,7 +11,7 @@ rm -rf build/ || true
 # Patch squashfuse_ll to be a library rather than an executable
 
 cd squashfuse
-patch -p1 -N < ../squashfuse.patch
+if [ ! -e ./ll.c.orig ]; then patch -p1 --backup < ../squashfuse.patch ; fi
```
Comment (probonopd) [inline build.sh]: good :-)

**Pull request merged**